src/main/resources/static/service_openapi.yaml

```diff
@@ -288,7 +288,7 @@ components:
         - DONE
         - ERROR
         - ABANDONED
-        - DUPLIATED
+        - DUPLICATED
```

Comment:
Sure fixing this won't break anything?

Reply:
Thinking about this more, Janitor is returning this to clients, so any old client version talking to the new server would probably have trouble parsing the enum. 
We could either make a breaking change or do a two part "introduce new enum, then delete the old one" migration. Either way, probably better to do that on it's own, PF-1628 for when someone needs a bite-sized change